app/src/main/res/xml/xdrip_plus_prefs.xml

```diff
@@ -684,19 +685,6 @@
                 android:title="@string/use_trend_momentum" />
 
         </PreferenceCategory>
-        <PreferenceCategory
-            android:key="title_I_understandb"
-            android:title="@string/eula">
-            <Preference
-                android:key="I_understandb"
-                android:summary="@string/not_for_medical_use"
-                android:title="@string/end_user_license_agreement">
-                <intent
-                    android:action="android.intent.action.MAIN"
-                    android:targetClass="com.eveningoutpost.dexdrip.LicenseAgreementActivity"
-                    android:targetPackage="@string/local_target_package" />
-            </Preference>
-        </PreferenceCategory>
```

Comment:
This extra EULA button has been moved. The Preference hasn't changed, but the PreferenceCategory surrounding it was removed.
It now lives in the "xdrip_version_category".